lib/ansible/playbook/play_context.py

```diff
@@ -422,7 +422,7 @@ def detect_su_prompt(data):
 
             elif self.become_method == 'pbrun':
 
-                prompt='assword:'
+                prompt='Password:'
```

Comment:
this was 'assword' on purpose as diff versions did not capitalize it the same way.